src/sqlfluff/core/dialects/__init__.py

```diff
@@ -36,6 +36,7 @@
     "teradata": ("dialect_teradata", "teradata_dialect"),
     "tsql": ("dialect_tsql", "tsql_dialect"),
     "sparksql": ("dialect_sparksql", "sparksql_dialect"),
+    "databricks": ("dialect_databricks", "databricks_dialect"),
```

Comment:
I wonder would it work to have this, so you don’t actually need the dialect files?

```suggestion
    "databricks": ("dialect_sparksql", "sparksql_dialect"),
```

Or do you see this not being a complete alias and having some differences in future?

Reply:
so this is how I initially implemented the alias but was getting failures with `/Users/lorin.dawson/repos/sqlfluff/test/cli/autocomplete_test.py`.

I think what you are suggesting is the better way to do this but couldn't figure out how to pass tests w/o changing the test case.

Reply:
Adding the dialect files was an easy way to get everything passing and start the review process :) but I agree that they're fundamentally not needed